centos/Dockerfile

```diff
@@ -3,6 +3,18 @@ LABEL maintainer="Kong Core Team <team-core@konghq.com>"
 
 ENV KONG_VERSION 0.15.0
 
+ARG SU_EXEC_VERSION=0.2
+ARG SU_EXEC_URL="https://github.com/ncopa/su-exec/archive/v${SU_EXEC_VERSION}.tar.gz"
+
+RUN yum install -y -q gcc make \
+&& curl -sL "${SU_EXEC_URL}" | tar -C /tmp -zxf - \ 
+ && make -C "/tmp/su-exec-${SU_EXEC_VERSION}" \
+&& cp "/tmp/su-exec-${SU_EXEC_VERSION}/su-exec" /usr/bin \
+&& rm -fr "/tmp/su-exec-${SU_EXEC_VERSION}" \
+&& yum autoremove -y -q gcc make \
```

Comment:
I'm not sure why but this doesn't actually remove `make`:
```
sh-4.2# yum list installed | grep make
make.x86_64                               1:3.82-23.el7                @base
```

Running the same command on a centos:7 docker image returns no results.

Reply:
lol I figured it out

```
yum install -y https://bintray.com/SNIP/kong-community-edition-0.15.0.el7.noarch.rpm

.....

Dependency Installed:
  make.x86_64 1:3.82-23.el7
  openssl.x86_64 1:1.0.2k-16.el7
```